tests/e2e/util.go

```diff
@@ -810,6 +810,27 @@ func invokeVCenterServiceControl(command, service, host string) error {
 	return nil
 }
 
+// waitVCenterServiceToStart invokes the status check for the given service and wait for it be running
+// via service-control on the given vCenter host over SSH.
+func waitVCenterServiceToStart(serviceName string, host string) error {
+	waitErr := wait.PollImmediate(poll, pollTimeoutShort, func() (bool, error) {
+		sshCmd := fmt.Sprintf("service-control --%s %s", "status", serviceName)
+		framework.Logf("Invoking command %v on vCenter host %v", sshCmd, host)
+		result, err := fssh.SSH(sshCmd, host, framework.TestContext.Provider)
+
+		if err != nil || result.Code != 0 {
+			fssh.LogResult(result)
+			return false, fmt.Errorf("couldn't execute command: %s on vCenter host: %v", sshCmd, err)
+		}
+		if strings.Contains(result.Stdout, "Running") {
+			framework.Logf("Found service Running %v", serviceName)
+			return true, nil
+		}
+		return false, nil
+	})
+	return waitErr
```

Comment:
add `fssh.LogResult(result)` before returning `waitErr`

Reply:
add `fssh.LogResult(result)` before returning `waitErr`, we should what was the state of service before we return `waitErr`

Reply:
the changes are not there in the latest diff, check again